test/openAPI/features/support/helpers/helpers.js

```diff
@@ -45,6 +45,49 @@ module.exports = {
       },
     ],
   },
+  //applicant_applications_fileId
+  applicationFileEndpoint: 'applications/{fileId}',
+  applicationFileResponseSchema: {
+    type: 'object',
+    properties: {
+      fileId: { type: 'string' },
+      registered: { type: 'string' },
+      serviceId: { type: 'string' },
+      serviceName: { type: 'string' },
+      status: {
+        type: 'object',
+        properties: {
+          code: {
+            type: 'string',
+            enum: [
+              'PENDING',
+              'APPROVED',
+              'SENDBACK',
+              'REJECTED'
+            ]
+          },
+          title: { type: 'string' }
+        }
+      },
+      ended: { type: 'boolean' },
+      applicationData: {
```

Comment:
This part of the response schema is used several times in the API spec, so good to add it in shares section and name it `applicationFileSubmit`